src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,98 @@
+import {
+  Container,
+  Row,
+  StyledButton,
+  BellWrapper,
+  Bell,
+  firstRow,
+} from './ButtonGroup.styles';
+import { FaBell } from 'react-icons/fa';
+
+const ButtonGroup = (props) => (
+  <Container>
+    <BellWrapper style={firstRow}>
+      <Bell>
+        <FaBell />
+      </Bell>
+    </BellWrapper>
+    <Row style={firstRow}>
+      <StyledButton big={true} bgColor="#77f4d6" fgColor="#003b2bff">
+        Buy / Sell &#x25BC;
+      </StyledButton>
+      <StyledButton big={true} bgColor="#77f4d6" fgColor="#003b2bff">
+        Long / Short &#x25BC;
+      </StyledButton>
+      <StyledButton big={true} bgColor="#77f4d6" fgColor="#003b2bff">
+        Earn / Loan &#x25BC;
+      </StyledButton>
+    </Row>
+    Market Cap
+    <Row>
+      <StyledButton big={false} fgColor="#003f3cff" bgColor="darkgrey">
+        Rank #{props.rank}
+      </StyledButton>
+    </Row>
+    Website
+    <Row>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Bitcoin.org
+      </StyledButton>
+    </Row>
+    Explorers
+    <Row>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Blockchair
+      </StyledButton>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Btc
+      </StyledButton>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Tokenview
+      </StyledButton>
+    </Row>
+    Wallets
+    <Row>
+      <StyledButton big={false} bgColor="#77f4d6" fgColor="#003b2bff">
+        Crypto.com DeFi Wallet
+      </StyledButton>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Ledger
+      </StyledButton>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Trezor
+      </StyledButton>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Electrum
+      </StyledButton>
+    </Row>
+    Community
+    <Row>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Reddit
+      </StyledButton>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Twitter
+      </StyledButton>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        Facebook
+      </StyledButton>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
+        bitcointalk.org
+      </StyledButton>
+    </Row>
+    Source Code
+    <Row>
+      <StyledButton big={false} bgColor="#003f3cff" fgColor="#77f4d6">
```

Comment:
put this color in a variable or use styled components and a prop

Reply:
I'm not sure what you mean by using styled components and a prop, as I'm already using a styled component and a prop, but I did put the colors in a variable.